xml/Microsoft.Build.Tasks/Csc.xml

```diff
@@ -94,7 +94,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets a Boolean value that indicates whether to compile code that uses the [unsafe](~/docs/csharp/language-reference/keywords/unsafe.md) keyword.</summary>
+        <summary>Gets or sets a Boolean value that indicates whether to compile code that uses the <see href="https://docs.microsoft.com/dotnet/csharp/language-reference/keywords/unsafe">unsafe</see> keyword.</summary>
```

Comment:
I'd keep the ~/docs links in order for us to get link validation at build time

Reply:
Do relative links work in the `<summary>` section now @mairaw? I thought previously they didn't, and had to be fully qualified?

Thanks for looking this over!

Reply:
Let me double check!

Reply:
Hey @mairaw - any update on this one? 😄

Reply:
Argh sorry @ChrisMaddock. Let me see if I can find a repro now.

Reply:
I found hrefs being used with the relative links but none are in summaries 😢 I know they work for docs, not sure if they work with the API browser and were still trying to solve some issues preventing us from indexing our APIs in the API browser right now, so it'd take a while to confirm. Let's use the docs,microsoft.com links now and we can switch to relative links once we can confirm.

Reply:
Great - sounds good to me! 😄